src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -17,6 +17,21 @@ export default (accessToken, userID, opts) => {
     identityVerificationLoader: gravityLoader(
       (id) => `identity_verification/${id}`
     ),
+    identityVerificationsLoader: gravityLoader(
+      "identity_verifications",
+      {},
+      { headers: true }
+    ),
+    identityVerificationOverridesLoader: gravityLoader(
+      (id) => `identity_verification/${id}/overrides`,
+      {},
+      { headers: true }
+    ),
+    identityVerificationScanReferencesLoader: gravityLoader(
+      (id) => `identity_verification/${id}/scan_references`,
+      {},
+      { headers: true }
+    ),
```

Comment:
@joeyAghion - out of curiosity is this effectively a rewrite of the Gravity GraphQL stitching code?

Reply:
Not that I know of... these are REST-ful list APIs that Gravity already exposes, but have never been exposed via GraphQL because Torque hit the API directly.